Sources/MySQL/Connection.swift

```diff
@@ -46,7 +46,31 @@ public final class Connection {
         
         mysql_set_character_set(cConnection, encoding)
     }
-
+    
+    public func transaction(_ closure: (Void) throws -> Void) throws {
```

Comment:
I think `()` syntax is used over `Void` or `(Void)` in the rest of Vapor's packages.